README.md

```diff
@@ -212,6 +212,7 @@ Waste collection schedules in the following formats and countries are supported.
 - [SSAM](/doc/source/ssam_se.md) / ssam.se
 - [Sysav Sophämntning](/doc/source/sysav_se.md) / sysav.se
 - [VA Syd Sophämntning](/doc/source/vasyd_se.md) / vasyd.se
+- [Landskrona - Svalövs Renhållning](/doc/source/lsr_nu.md) / lsr.nu
```

Comment:
Title in README.md differs from TITLE field in lsr_nu.py --> please correct and use update_docu_links.py to generate the linkss.
Otherwise the title in README.md (and info.md) will be overwritten the next time s.o. uses update_docu_links.py

Reply:
Sorry, completely missed that script. Updated with new COUNTRY setting in the source file and having run the script now. Apologies for the unsquashed commit history.